example/android/app/src/androidTest/java/com/revenuecat/purchases_flutter/MainActivityTest.java

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases_flutter;
+
+import androidx.test.rule.ActivityTestRule;
+import dev.flutter.plugins.e2e.FlutterRunner;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+import io.flutter.embedding.android.FlutterActivity;
+
+@RunWith(FlutterRunner.class)
+public class MainActivityTest {
+    @Rule
+    public ActivityTestRule<FlutterActivity> rule = new ActivityTestRule<>(FlutterActivity.class);
+}
```

Comment:
This test makes sure the new APIs are supported and the plugin is registered successfully. I copied this from migration docs https://flutter.dev/docs/development/packages-and-plugins/plugin-api-migration#testing-your-plugin